### PR TITLE
site web responsive

### PR DESCRIPTION
**Pull request by PriscilliaAmmeux**

### CODE REVIEW

**Review by Azahires — DISMISSED** [review]

File: frontend/src/components/Location/Location.jsx [hunk 1]
```diff
@@ -1,15 +1,14 @@
 import "./location.css";
-import { PropTypes } from "prop-types";
+// import { PropTypes } from "prop-types";
+// import { useEffect, useState } from "react";
+// import axios from "axios";
```
Comment (LoicBrassart) [inline frontend/src/components/Location/Location.jsx]: dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
**Almost** no Evil

**Review by CharlyJuteau — APPROVED** [review]

**Review by CharlyJuteau — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -18,5 +18,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "axios": "^1.1.3"
```
Comment (LoicBrassart) [inline package.json]: Nope !

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Why, ô why ? é_è

**Review by CharlyJuteau — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
Better

**Review by Azahires — APPROVED** [review]
Saw no evil as said a great viking !

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**